test-packages/integration-tests/test/test-data/batch/changeset-request.ts

```diff
@@ -3,14 +3,13 @@ import {
   testEntityKeyPropString,
   testEntityMultiLinkKeyProp
 } from '../keys';
-import { changesetHeader, changesetRequestHeader } from './header';
+import { changesetHeader, requestHeader } from './header';
```

Comment:
General: We have also a folder test-resources. Perhaps it makes sense to put the test data there? I am not sure.